projects/vdk-plugins/vdk-impala/src/vdk/plugin/impala/templates/load/fact/insert-snapshot/README.md

```diff
@@ -0,0 +1,51 @@
+### Purpose:
+
+This template can be used to load raw data from Data Lake to target 'Snapshot Periodic Fact Table' in Data Warehouse.
+In summary, it appends a snapshot of records observed between time t1 and t2 from the source table to the target table. After that inserts the rows from the source table.
+
+### Details:
+
+<https://www.kimballgroup.com/data-warehouse-business-intelligence-resources/kimball-techniques/dimensional-modeling-techniques/periodic-snapshot-fact-table/>
+
+### Template Name (template_name):
+
+- "periodic_snapshot"
+
+### Template Parameters (template_args):
+
+- target_schema   - SC Data Warehouse schema, where target data is loaded
+- target_table    - SC Data Warehouse table of DW type 'Snapshot Periodic Fact Table', where target data is loaded
+- source_schema   - SC Data Lake schema, where source raw data is loaded from
+- source_view     - SC Data Lake view, where source raw data is loaded from
+- last_arrival_ts - Timestamp column, on which increments to target_table are done
```

Comment:
I think last_arrival_ts is not needed here. It's not used anywhere in the tempalte.